src/pacts/frontend-app-profile-edx-platform.json

```diff
@@ -0,0 +1,117 @@
+{
```

Comment:
I'm guessing this file is an unintended duplicate?

Reply:
Yeah, seems like it generated two pacts: One located in the src folder and one in the entire directory.

Reply:
The files are a bit different, with the `staff_not_found` interaction having moved, and a space character having snuck into one of the URLs.

Reply:
(`src/pacts/frontend-app-profile-edx-platform.json` was the one with the space char, so that's all good now)

Reply:
Should i move back the ```staff_not_found``` interaction back? The file was auto-generated from the pact package so a bit confused on that.